App.tsx

```diff
@@ -9,8 +11,12 @@ import Navigation from "./navigation";
 export default function App() {
   const isLoadingComplete = useCachedResources();
   const colorScheme = useColorScheme();
+  const [loaded] = useFonts({
+    OleoScript: require("./assets/fonts/OleoScript-Regular.ttf"),
+    Caramel: require("./assets/fonts/Caramel-Regular.ttf"),
+  });
 
-  if (!isLoadingComplete) {
+  if (!isLoadingComplete && !loaded) {
     return null;
```

Comment:
Maybe this should return an ActivityIndicatior instead of null?

Reply:
Ohhh that would be nice, will add it!